api/v2/models/politico.py

```diff
@@ -193,3 +194,28 @@ def login(self, email, password):
 
         user = User.login(email, password)
         return user
+
+    def create_candidate(self, user_id, office_id):
+        if user_id is None:
+            raise InputError('User id is required')
+
+        if office_id is None:
+            raise InputError('Office id is required')
+
+        candidate = Candidate.save_candidate(user_id, office_id)
+        return candidate
+
+    def create_vote(self, candidate_id, office_id, voter_id):
```

Comment:
The PR description says that this PR is about implementing a candidate feature, yet here I can see that you are creating a vote. Kindly add the this on your PR description as well as a link to your PT story about this task.